src/main/java/io/tus/java/client/TusClient.java

```diff
@@ -144,10 +144,18 @@ public void setHeaders(@Nullable Map<String, String> headers) {
         return headers;
     }
 
+    /**
+     * Sets the timeout for a Connection.
+     * @param timeout in seconds //todo is this seconds?
```

Comment:
Are we talking about seconds or another unit?

Reply:
It uses milliseconds: https://docs.oracle.com/javase/8/docs/api/java/net/URLConnection.html#setConnectTimeout-int-